File: Level_1/핸드폰 번호 가리기/phone_Number.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(const char* phone_number) {
    char* answer = (char*)malloc(sizeof(char)*strlen(phone_number) + 1);
    strcpy(answer, phone_number);
    int p_len = strlen(phone_number);
    for(int i = 0; i < p_len - 4; i++){
       answer[i] = '*';
    }
    return answer;
}

File: Level_1/핸드폰 번호 가리기/phone_Number.py
def solution(phone_number):
    answer = ''
    p_len = len(phone_number)
    
    for i in range(0, p_len) :
        if i < p_len - 4 :
            answer += '*'
        else :
            answer += phone_number[i]   
            
    return answer
